index.js

```diff
@@ -40,7 +40,8 @@ var browserInfo = null;
 // Aggregates all test results by test name, e.g. allTestResultsByKey['angrybots'] is an array containing results of each run of that demo.
 var allTestResultsByKey = {};
 
-var testData = await fetch(`${window.location.href}/demo_list.json`);
+var host = window.location.protocol + "//" + window.location.host;
+var testData = await fetch(`${host}/demo_list.json`);
```

Comment:
Nit: Any reason not to make this a one-liner? I prefer to see the full URL template where possible.

```javascript
var testData = await fetch(`${window.location.protocol}//${window.location.host}/demo_list.json`);
```

Reply:
I actually prefer not to have one-liners because I personally think its easier to read as opposed one long single line and find it easier to debug when there are different variables assigned to things. But in this case I think either way is fine if you'd like me to change it